plate_mapper/plate_mapper.py

```diff
@@ -68,22 +96,62 @@ def plate_mapper(input_f, barseq_f, output_f):
     barseq_f.close()
     print('  Done.')
     # Write output sequencing run file
+    samples = {}
```

Comment:
Got to the end of all this block and realized that you are basically counting the sample names, you could do that by doing something like:
```
In [1]: from collections import Counter
In [2]: Counter([1, 2, 2, 3, 5, 3, 'aaa', 'aaa', 'bb'])
Out[2]: Counter({2: 2, 3: 2, 'aaa': 2, 1: 1, 5: 1, 'bb': 1})
```